README.md

```diff
@@ -72,3 +72,28 @@ TBD
 ### Trim (Planned)
 
 TBD
+
+## Verifying Releases
+
+Bomctl releases can be [found here](https://github.com/bomctl/bomctl/releases) and are signed
+using keyless signing with cosign.
+
+For each artifact there are two additional files:
+
+- `${artifact}-keyless.sig` - signature
+- `${artifact}-keyless.pem` - certificate
+
+You can then verify this artifact with cosign the signature and the certificate.
```

Comment:
You can then verify this artifact with cosign **using** the signature and the certificate.